clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/IsolatedIngestionBackend.java

```diff
@@ -25,7 +26,7 @@
 
 
 /**
- * IsolatedIngestionBackend is the implementation of ingestion backend designed for ingestion isolation.
```

Comment:
Nit: Maybe {@link IsolatedIngestionBackend} instead of put raw class name in java doc?